plugins/postgres/dbt/include/postgres/macros/materializations/materialized_view.sql

```diff
@@ -27,11 +42,28 @@
 {% endmacro %}
 
 
-{% macro default__get_materialized_view_configuration_changes(existing_relation, new_config) %}
-    {%- do return([]) -%}
+{% macro postgres__get_materialized_view_configuration_changes(existing_relation, new_config) %}
```

Comment:
@mikealfare curious to what ideas you have for this would you want to move it to a impl method?